packages/core/src/card/InteractableCardGroup.css

```diff
@@ -0,0 +1,6 @@
+.saltInteractableCardGroup {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: calc(var(--salt-size-unit) * 2);
```

Comment:
More of a general question on when we use --salt-size-unit and --salt-spacing variables ?
@joshwooding

Reply:
unit is deprecated, should be using spacing-100